bindings/rascal/models/IP_generic_md.py

```diff
@@ -0,0 +1,103 @@
+import ase.io
```

Comment:
This file name is a bit weird. What does IP mean here? Interatomic Potential? How about moving this to `bindings/rascal/models/md.py`? This already implies an interatomic potential IMO.

Reply:
It was initially for consistency with `IP_ase_interface.py`, but I wouldn't mind a shorter module name.

Reply:
I think the same argument applies for `IP_ase_interface.py` to be renamed to `ase_interface.py`/`ase_driver.py`